spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/VaultKvAccessStrategy.java

```diff
@@ -36,77 +32,11 @@
 	/**
 	 * Return secrets from Vault. The response is represented as JSON object marshaled to
 	 * {@link String}.
-	 * @param headers must not be {@literal null}.
-	 * @param backend secret backend mount path, must not be {@literal null}.
 	 * @param key key within the key-value secret backend, must not be {@literal null}.
 	 * @return the marshaled JSON object or {@literal null} if the key was not found.
 	 * @throws RestClientException in case of a transport/access failure.
 	 * @see com.fasterxml.jackson.annotation.JsonRawValue
 	 */
-	String getData(HttpHeaders headers, String backend, String key)
```

Comment:
Maybe default method that calls `getData(key)`?

Reply:
Resolved in 9c2993cfb16697b7702d374ebe14d6a435cc9d5c